config/dm_config/nnf-dm-config.yaml

```diff
@@ -1,6 +1,30 @@
+# Each profile is capable of providing different configurations for data movement.
+# Note: For now, only the default profile is supported.
 profiles:
+
+  # Default profile that is used for all data movement activity.
   default:
+      # The number of slots to be use in the MPI hostfile. A value less than 1 disables the use
```

Comment:
"to be use" => "to use"?  Or should it be "to specify"?

And again below for maxSlots.